components/pricing/Tier.vue

```diff
@@ -52,7 +52,18 @@ const props = defineProps({
   popular: Boolean,
   discounted: Boolean,
 })
+const country = useCookie('front_currency');
```

Comment:
Any way to make this a plugin/middleware/something so that we don't copy-paste the code 3 times and have to update 3 variables when a new country switches to the EUR?